docs/scenarios/actions.rst

```diff
@@ -482,6 +482,24 @@ and microseconds since the epoch. For example::
 
 
 
+urlencode / urldecode
++++++++++++++++++++++
+
+The urlencode and urldecode actions will replace the content of the
+variable specified in assign_to with the coded version. For example::
+
+Content of variable2 is "this: is a string".
+
+    <nop>
+      <action>
+        <urlencode assign_to="variable2" />
```

Comment:
This PR looks awfully complete :+1: 

I just wonder about this `assign_to`. Doesn't it make more sense to call it `variable`?
```xml
<urlencode variable="variable2" />
```

Reply:
Absolutely, I based my work on the trim method, that explains why I used assign_to :)
I'll give it a quick change.